widgets/src/lib/InferenceWidget/InferenceWidget.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
 	import type { SvelteComponent } from "svelte";
-	import type { PipelineType } from "../../../../interfaces/Types";
+	import type { PipelineType } from "../interfaces/Types";
```

Comment:
I think we can use `$lib/interfaces/...` everywhere now in the widgets instead no?

Reply:
Updated all the imports to use `$lib/interfaces/...` 👍